docs/posts/signal-number-registration-update.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2022-11-10
+categories:
+    - Software
+authors:
+    - matchboxbananasynergy
+tags:
+    - PSA
+    - Signal
+    - Instant Messengers
+links:
+    - Signal Configuration Guide: https://www.privacyguides.org/real-time-communication/signal-configuration-hardening/
+    - Real-Time Communication: https://www.privacyguides.org/real-time-communication/
+license: BY-SA
+---
+# Important Changes to Signal Registration and Registration Lock
+
+There has been a recent change to how Signal handles registration which primarily affects people who are using a number that they don't have exclusive access to for Signal.
+
+## How It Used to Work
+
+As outlined in our previous [Signal Configuration & Hardening Guide](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening/#signal-pin), if you register a number with Signal, set up Registration Lock and check into the app at least once every 7 days, nobody could use the number you've claimed and try to re-register it for themselves without knowing your Registration Lock PIN.
```

Comment:
In our [previous](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening/#signal-pin) guide we outlined how to prevent people re-registering your number for Signal using the Registration Lock feature.